python/tests/utils/test_manifest.py

```diff
@@ -253,3 +265,14 @@ def test_read_manifest_v2(generated_manifest_file_file_v2: str) -> None:
     assert partition.contains_nan is False
     assert partition.lower_bound == b"\x01\x00\x00\x00"
     assert partition.upper_bound == b"\x02\x00\x00\x00"
+
+    entries = manifest_list.fetch_manifest_entry(io)
```

Comment:
How is the manifest list written? It looks like the location is static in the test fixtures.